src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Agricultoras.kt

```diff
@@ -0,0 +1,19 @@
+package ar.edu.unahur.obj2.semillasAlViento
+
+class Agricultora(val parcelas: MutableList<Parcela>) {
+
+    fun parcelasSemilleras() =
+        parcelas.filter {
+                parcela -> parcela.plantas.all { planta -> planta.daSemillas() }
+        }
```

Comment:
Esta lógica interna la podrías haber hecho en la parcela
```suggestion
    fun parcelasSemilleras() = parcelas.filter { parcela -> parcela.esSemillera() }
```